### PR TITLE
Return user from init and small event fix

### PR DESCRIPTION
**Pull request by lemieux**
@mspensieri 

as discussed, this returns the app user when you either call `init` or `login`, (and `logout` for that matter).

Also fixed a bug in the events for messages.

### CODE REVIEW

File: src/js/smooch.jsx [hunk 1]
```diff
@@ -190,6 +190,8 @@ export class Smooch extends Observable {
                 this._el = renderWidget();
             }
             this.trigger('ready');
+
+            return store.getState().user;
```
Comment (mspensieri) [inline src/js/smooch.jsx]: Should we also pass this to the `ready` callback? Is it possible? `this.trigger('ready', store.getState().user)`

**Comment by mspensieri** [comment]
:+1:

File: src/js/smooch.jsx (hunk 1 above)
Comment (lemieux) [inline src/js/smooch.jsx]: yes we should

**Comment by jugarrit** [comment]
:+1:

**Pull request merged**